next/src/errors/messages.ts

```diff
@@ -81,17 +81,17 @@ export function getErrorMessage(
     }
     // Arrays
     case 'minItems':
-      throw new Error('Array support is not implemented yet')
```

Comment:
todo: We forgot something in these error messages, based [on this convo](https://linear.app/remote/issue/RMT-51/jsf-support-group-array-to-have-visibility-for-conditional-properties#comment-686a7c8d). So although we don't support hiding a conditional field, at least we show a better error message. "Not allowed" is more intuitive than "Always fails", would you agree?

```diff
    case 'valid':
-      return 'Always fails'
-      return 'Not allowed'
```

P.S. This case being called `valid` is so weird, A better name could be `forbidden` or something, as this is a direct match of a schema like `{ address: false }`

Reply:
💯 I've changed it now.

Reply:
> Why do you think forbidden makes assumption about the presentation ui?

You know what? I have no good argument now that I think about it 😅 . Let's do `"forbidden"`. It does make sense, when a value is provided for this schema it fails.